sdk/core/core-http/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.0.4 (Unreleased)
 
+- Updated the error thrown when a request is terminated with `OperationRequestOptions.timeout` in browsers. An abort error is thrown with the message - "The operation was aborted." if the request is terminated. [PR #7159](https://github.com/Azure/azure-sdk-for-js/pull/7159)
```

Comment:
```suggestion
- When an operation is terminated due to it getting timed out based on the `timeout` configured in the `OperationRequestOptions`, the error that is thrown in browser is now updated to match what is thrown in node. In these cases, an `AbortError` is thrown instead of the previous `RestError`. [PR #7159](https://github.com/Azure/azure-sdk-for-js/pull/7159)
```

Reply:
Nice, thanks. Updated.